joern-cli/frontends/csharpsrc2cpg/src/test/scala/io/joern/csharpsrc2cpg/querying/dataflow/MethodsAndCallDataflowsTests.scala

```diff
@@ -89,4 +89,70 @@ class MethodsAndCallDataflowsTests extends CSharpCode2CpgFixture(withDataFlow =
     }
   }
 
+  "dataflow for methods and calls across files with a file scoped namespace" should {
+    val cpg = code(
+      """
+        |using HelloBaz.Bazz.Baz;
+        |
+        |namespace HelloWorld {
+        |public class Foo {
+        | public int bar(int pBar) {
+        |   var b = new Baz();
+        |   int res = b.qux(1, "hello");
+        |   return res;
+        | }
+        |}
+        |}
+        |""".stripMargin,
+      "Foo.cs"
+    ).moreCode(
+      """
+        |namespace HelloBaz.Bazz;
+        |public class Baz {
+        | public int qux(int pQux, string ppQux) {
+        |   return pQux;
+        | }
+        |}
+        |""".stripMargin,
+      fileName = "Baz.cs"
+    )
+
+    "find a path from 1 to res (case 2)" in {
+      val src  = cpg.literal.codeExact("1").l
```

Comment:
This could use the `inside()` pattern, but not sure what we decided on finally.

Reply:
We don't write dataflow tests using `inside()`